docs/UserGuide.md

```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
 
 To create a deadline named `EE2024 homework 1` at 2016 1st October 6 am.
 
+### List tasks: `list`
+
+List certain type of task stored in the database
```

Comment:
I think this command should only be finalised when we finalise our UI design, otherwise, we may not be able to decide on how this command really works.
Since for our first design, deadlines and floating tasks are already listed on the UI